usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -803,9 +1057,19 @@ def test_load_table_to_from_delta_for_award_search_testing(
 
 @mark.django_db(transaction=True)
 def test_load_table_to_from_delta_for_award_search_alt_db_and_name(
-    spark, s3_unittest_data_bucket, populate_data_for_transaction_search
+    spark, s3_unittest_data_bucket, populate_data_for_transaction_search, hive_unittest_metastore_db
 ):
-    create_and_load_all_delta_tables(spark, s3_unittest_data_bucket)
+    tables_to_load = [
+        "awards",
+        "financial_accounts_by_awards",
+        "recipient_lookup",
```

Comment:
For this, and other places where `create_and_load_all_delta_tables` is called, just take a look at probably next-sprint refactoring considering this comment: https://github.com/fedspendingtransparency/usaspending-api/pull/3539#discussion_r944792655